bootstrap4/.js/src/main/scala/io/udash/bootstrap/dropdown/UdashDropdown.scala

```diff
@@ -78,7 +78,7 @@ final class UdashDropdown[ItemType, ElemType <: ReadableProperty[ItemType]] priv
         nestedInterceptor(BootstrapStyles.Dropdown.menuRight.styleIf(rightAlignMenu)),
         aria.labelledby := buttonId, id := menuId
       )(nestedInterceptor(
-        produce(items.transform((item: ItemType) => withSelectionListener(itemFactory(item), item)))(el => el)
+        produce(items.transform((item: ItemType) => withSelectionListener(itemFactory(item, nestedInterceptor), item)))(el => el)
```

Comment:
You should use `produceWithNested` to properly clean up bindings in case of `items` changes,